### PR TITLE
Feature - Manage schedules (3/3)

### PR DESCRIPTION
**Pull request by sasirven**
This is the GUI part of the schedules management with cron, which depends on the https://github.com/mitre/caldera/pull/3025 and https://github.com/mitre/caldera/pull/3026

Linked Merge Requests:
https://github.com/mitre/caldera/pull/3025 
https://github.com/mitre/caldera/pull/3026

- Adds a schedule view:
  - Accessible from the navigation menu,
  - Handles schedule creation / edition / deletion

### CODE REVIEW

File: src/components/schedules/CreateScheduleModal.vue [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+            label.label.ml-3.mt-1(for="manual") Require manual approval
+      .field.is-horizontal
+        .field-label
+          label.label Parser
+        .field-body
+          .field.is-grouped
+            input.is-checkradio(type="radio" id="defaultparser" :checked="isDefParser" @click="isDefParser = true" :disabled="scheduleStore.currentSchedule")
+            label.label.ml-3.mt-1(for="defaultparser") Use Default Parser
+            input.is-checkradio.ml-3(type="radio" id="nondefaultparser" :checked="!isDefParser" @click="isDefParser = false" :disabled="scheduleStore.currentSchedule")
+            label.label.ml-3.mt-1(for="nondefaultparser") Require manual approval
```
Comment (uruwhy) [inline src/components/schedules/CreateScheduleModal.vue:316]:
```suggestion
            label.label.ml-3.mt-1(for="nondefaultparser") Don't use default learning parsers
```

Comment (uruwhy) [inline src/components/schedules/CreateScheduleModal.vue:316]:
```suggestion
            label.label.ml-3.mt-1(for="nondefaultparser") Don't use default learning parsers
```

**Comment by sasirven** [comment]
Hello @uruwhy can you check on this MR?
It's the final part of the schedules management part 😄

**Review by uruwhy — APPROVED** [review]

**Comment by elegantmoose** [comment]
Testing now. So far looks great!

**Comment by elegantmoose** [comment]
This LGTM. I just would like the default string hint for crontab format.

**Review by elegantmoose — APPROVED** [review]

**Comment by sasirven** [comment]
> This LGTM. I just would like the default string hint for crontab format.

Hello sorry for the delay and thanks for the commit 😅

**Pull request merged**